addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2080,8 +2084,8 @@ const ListUserValueWidget = UserValueWidget.extend({
                 return;
             }
             prepare();
-            const { id, display_name } = JSON.parse(widget.getMethodsParams('addRecord').recordData);
-            this._addItemToTable(id, display_name);
+            const { id, display_name, ...extra } = JSON.parse(widget.getMethodsParams('addRecord').recordData);
```

Comment:
I don't really like to use this while it is still marked as an experimental feature: https://caniuse.com/mdn-javascript_operators_destructuring_rest_in_objects